src/routes/NoteCreateRoute.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import '../../test/jest/__mock__/currencyData.mock';
+import '../../test/jest/__mock__/stripesConfig.mock';
+import { Router } from 'react-router';
+import { createMemoryHistory } from 'history';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import NoteCreateRoute from './NoteCreateRoute';
+
+jest.mock('@folio/stripes/smart-components', () => ({
+  ...jest.requireActual('@folio/stripes/smart-components'),
+  NoteCreatePage: (props) => <div><span>NoteCreatePage</span><button type="button" onClick={() => props.renderReferredRecord()}>renderReferredRecord</button></div>,
```

Comment:
Could you add mock to 'ui-requests\test\jest\__mock__\stripesSmartComponents.mock.js'?